oaknational/Oak-Web-Application
# feat/1315 add disabled prop to brush border button

## Description

add disabled prop to brush border button (so other button types are not affected)

## Issue(s)

Fixes #1315

## How to test

1. check button in story book
2. check that buttons on About us page are not affected (I will verify that in Percy too)
